css-forms-1/Overview.bs

```diff
@@ -695,6 +695,11 @@ input[type=radio]::checkmark {
 ## Selects & buttons
 
 ```css
+select {
+  /* Base appearance select always sizes based on its contents. */
+  field-sizing: content !important;
```

Comment:
We maybe only want this for dropdown select and not for listbox select but we can handle that once we have a selector to choose between them.